src/cmps/button/Button.tsx

```diff
@@ -1,26 +1,31 @@
-//Button component
-
-import React, { MouseEvent } from "react";
+import React, { MouseEvent } from 'react';
 import './Button.css';
 
 // without state use react stateless function
 type props = {
     isDisabled?: boolean;
     value: string;
     onClick(e: MouseEvent<HTMLElement>): void;
-}
+};
 
 const Button: React.FC<props> = ({ isDisabled, value, onClick: handleClick }) => {
     var styleDisable: string = '';
-    if (isDisabled) { styleDisable = 'default' } else { styleDisable = 'disabled' };
+    if (isDisabled) {
+        styleDisable = 'default';
+    } else {
+        styleDisable = 'disabled';
+    }
     const classes: string = `button ${value} ${styleDisable}`;
+    /*eslint no-console: ["error", { allow: ["warn", "error", "log"] }] */
```

Comment:
avoid!

Reply:
I did not find another solution on the net...